.semaphore/semaphore.yml

```diff
@@ -0,0 +1,246 @@
+version: v1.0
+name: CI Pipeline

```

Comment:
The previous CI run took `23.20m` yet the current CI runs on the master take around `24m`. Shouldn't we be getting faster times?

Reply:
Only 3 processes are being run at any instance on the free tier which slows down the whole process. If we had all parallel processes running at the same time concurrently we would have faster times of ~19 min.